pcsx2/gui/Dialogs/AboutBoxDialog.cpp

```diff
@@ -37,69 +37,55 @@ Dialogs::AboutBoxDialog::AboutBoxDialog(wxWindow* parent)
 		wxDefaultPosition, wxDefaultSize
 		)
 {
-	// [TODO] : About box should be upgraded to use scrollable read-only text boxes.
-
-	wxString developsString = wxsFormat(
-		L"avih, Refraction, rama, pseudonym, gregory.hainaut, turtleli"
-		L"\n\n"
-		L"%s: \n"
-		L"Arcum42, Aumatt, drk||raziel, "
-		L"cottonvibes, gigaherz, saqib, "
-		L"\n"
-		L"Alexey Silinov, Florin, "
-		L"goldfinger, Linuzappz, loser, "
-		L"Nachbrenner, shadow, Zerofrog, tmkk, Jake.Stine"
-		L"\n\n"
-		L"%s:\n"
-		L"Falcon4ever, Bositman, Akasha",
-		_("Previous versions"), _("Webmasters"));
-
-	wxString contribsString = wxsFormat(
-		L"ssakash, FlatOut, CK1, lightningterror, atomic83github, Pandubz, Prafull, Blyss Sarania, Nobbs66, Alessandro Vetere"
-		L"\n\n"
-		L"%s: \n"
-		L"ChickenLiver (Lilypad), Gabest (GSdx, Cdvdolio, Xpad), Ckemu, "
-		L"General Plot, KrossX, Devina, micove, black_wd, Belmont, "
-		L"\n"
-		L"BGome, _Demo_, Dreamtime, Hiryu and Sjeep, nneeve, F|RES, "
-		L"Shadow Lady, MrBrown, razorblade, Seta-san, Skarmeth, feal87, Athos"
-		L"\n",
-		_("Previous versions"));
-
-	wxFlexGridSizer& boxesContainer = *new wxFlexGridSizer(2, 0, StdPadding);
-	boxesContainer.AddGrowableCol(0, 1);
-	boxesContainer.AddGrowableCol(1, 1);
-
-	wxStaticBoxSizer& developsBox = *new wxStaticBoxSizer(wxVERTICAL, this);
-	wxStaticBoxSizer& contribsBox = *new wxStaticBoxSizer(wxVERTICAL, this);
-
-	pxStaticText& developsText = Text(developsString).SetMinWidth(350);
-	pxStaticText& contribsText = Text(contribsString).SetMinWidth(350);
-
-	developsBox += Heading(_("Developers")).Bold() | StdExpand();
-	developsBox += developsText | StdExpand();
-	contribsBox += Heading(_("Contributors")).Bold() | StdExpand();
-	contribsBox += contribsText | StdExpand();
-
-	boxesContainer += developsBox | StdExpand();
-	boxesContainer += contribsBox | StdExpand();
-
 	// Main layout
+
+	SetMinWidth(460);
+
 	*this += m_bitmap_logo | StdCenter();
 
-	*this += Text(_("PlayStation 2 Emulator"));
+	*this += Text(_("PlayStation 2 Emulator:"));
+
+	*this += new wxHyperlinkCtrl(this, wxID_ANY,
+		_("Main Website"), L"https://pcsx2.net"
+	) | pxProportion(1).Center().Border(wxALL, 3);
+
+	*this += new wxHyperlinkCtrl(this, wxID_ANY,
+		_("Support Forums"), L"https://forums.pcsx2.net"
+	) | pxProportion(1).Center().Border(wxALL, 3);
+
+	*this += new wxHyperlinkCtrl(this, wxID_ANY,
+		_("Git Repository at GitHub"), L"https://github.com/PCSX2/pcsx2"
+	) | pxProportion(1).Center().Border(wxALL, 3);
+
+	*this += new wxHyperlinkCtrl(this, wxID_ANY,
+		_("Contributors for v1.6 release"), L"https://github.com/PCSX2/pcsx2/graphs/contributors?from=2016-01-08&to=2019-11-07&type=c"
```

Comment:
Link will need to be updated again during release date.

Reply:
Do we need to do by release? I think the point of this was to not have to update it all the time. Maybe just the non-date-ranged link and just call it "Contributors" or w/e.

Reply:
https://github.com/PCSX2/pcsx2/pull/3059#discussion_r327926696

Reply:
We do mention previous contributors tho.

Reply:
Sorry not following, what does using github date range have to do with google?

Reply:
Nothing really, idea was just to list active contributors for 1.6 release, just trying various things to see which people prefer the most.

Reply:
I say just link to the contributor list. If you have to have an additional link or some 3rd party side for google people then so be it. I just think it defeats the purpose Li set out with this to not have to update it every release if we make it a range that has to be updated every release.